assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -208,6 +326,18 @@ export const selectors = {
 
 		return error || null;
 	},
+
+	/**
+	 * Indicates whether account creation is currently in progress.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {boolean} True if an account is being created, false otherwise.
+	 */
+	isDoingCreateAccount( state ) {
+		return state.isCreatingAccount;
```

Comment:
This should be initialized in the `INITIAL_STATE` and be called `isFetchingCreateAccount`

Also, we `!!` the return value to guarantee it always returns a boolean.

Reply:
Added in https://github.com/google/site-kit-wp/pull/1390/commits/15a35872b4625c2ebde23c2ca1241033b7158687.